src/pages/networks/forms/NetworkDefaultACLSelector.tsx

```diff
@@ -0,0 +1,131 @@
+import type { FC, ReactNode } from "react";
+import { Fragment, useEffect, useState } from "react";
+import { CheckboxInput, Select } from "@canonical/react-components";
+
+type Direction = "Ingress" | "Egress";
+
+export interface DefaultACLSelectOption {
+  trafficDirection: string;
+  values: string[];
+}
+
+interface Props {
+  id: string;
+  className?: string;
+  isReadOnly: boolean;
+  onChange: (fieldValue: string, value: string) => void;
+  values?: Record<string, string>;
+  canToggleSpecific?: boolean;
```

Comment:
```suggestion
```
I think this will always be true, I don't see a use case for disabling this, so maybe can be removed?